app/celery/process_ga4_measurement_tasks.py

```diff
@@ -90,7 +101,23 @@ def post_to_ga4(
         'Content-Type': 'application/json',
     }
     current_app.logger.debug('Posting to GA4: %s', event_body)
-    response = requests.post(url, json=event_body, headers=headers, timeout=1)
-    current_app.logger.debug('GA4 response: %s', response.status_code)
 
-    return response.status_code == 204
+    status = False
+    try:
+        current_app.logger.info('Posting event to GA4: %s', name)
+        response = requests.post(url, json=event_body, headers=headers, timeout=1)
+        current_app.logger.debug('GA4 response: %s', response.status_code)
+        response.raise_for_status()
+        status = response.status_code == 204
+    except requests.HTTPError as e:
+        current_app.logger.error('GA4 HTTPError: %s', e)
+        raise AutoRetryException('GA4 HTTPError') from e
+    except requests.Timeout as e:
+        current_app.logger.error('GA4 Timeout: %s', e)
+        raise AutoRetryException('GA4 Timeout') from e
+    except requests.ConnectionError as e:
+        current_app.logger.error('GA4 ConnectionError: %s', e)
+        raise AutoRetryException('GA4 ConnectionError') from e
```

Comment:
This will work, so I don't insist upon a change, but it strikes me as verbose given that we agreed to bring back "exception" level logging (which is "error" with a proper traceback that includes the exception type).
```suggestion
    except (requests.HTTPError, requests.Timeout, requests.ConnectionError) as e:
        current_app.logger.exception(e)
        raise AutoRetryException from e
```